lib/temporal/workflow/context.rb

```diff
@@ -17,6 +17,24 @@ class Workflow
     class Context
       attr_reader :metadata, :config
 
+      module Private
+        # Shared with LocalWorkflowContext so we can do the same validations in test and production.
+        class Validators
```

Comment:
nit: since it's not expected to get initialised, we should probably just use `module`